source/manual/pentests.html.md

```diff
@@ -6,6 +6,6 @@ layout: manual_layout
 parent: "/manual.html"
 ---
 
-GOV.UK periodically runs penetration tests (otherwise known as 'pentests' or 'IT healthchecks') on its infrastructure, to identify security issues. Read more at [govuk-ithc-documentation][].
+GOV.UK periodically runs penetration tests (otherwise known as 'pentests' or 'IT healthchecks') on its infrastructure, to identify security issues. Any relevant documentation lives in the respective app repo or, if sensitive, in the [ITHC Google doc][].
 
-[govuk-ithc-documentation]: https://github.com/alphagov/govuk-ithc-documentation
+[ITHC Google doc]: https://docs.google.com/document/d/1npYap4f7i3qZulhqdlu0jAGMP_TFMHXX7TFLiM04UTw
```

Comment:
I'd say something like "the ITHC Google doc linked from the repo", rather than link the URL here.
Not the end of the world if this slips in, but it would be safer to not link directly to the resource.
Thanks!